jobs/config.json

```diff
@@ -8519,6 +8519,27 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7-upgrade-cluster-init": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=ci/latest-1.6",
+      "--gcp-zone=us-central1-f",
+      "--kubeadm=ci",
+      "--kubernetes-anywhere-kubernetes-version=latest-1.6",
+      "--kubernetes-anywhere-upgrade-provider=kubeadm",
+      "--kubernetes-anywhere-kubeadm-upgrade-method=init",
+      "--provider=kubernetes-anywhere",
+      "--test_args=--ginkgo.focus=\\[Conformance\\] --minStartupPods=8",
+      "--upgrade_args=--ginkgo.focus=\\[Feature:ClusterUpgrade\\] --upgrade-target=ci/latest-1.7",
```

Comment:
For the minimal viable product, we are only covering upgrading the control plane. Therefore we should do Feature:MasterUpgrade instead of Feature:ClusterUpgrade.

Reply:
Location of the *Upgrade features in case you are interested: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/lifecycle/cluster_upgrade.go#L81